templates/test/dev/custom-builds-load/patches/azuremachinetemplate-monitoring.yaml

```diff
@@ -1,6 +1,17 @@
 ---
 apiVersion: infrastructure.cluster.x-k8s.io/v1beta1
 kind: AzureMachineTemplate
+metadata:
+  name: ${CLUSTER_NAME}-control-plane
+  namespace: default
+spec:
+  template:
+    spec:
+      additionalTags:
+        monitoring: load
```

Comment:
Reconfirming; Does this mean we are trying to tag and monitor the control-plane nodes too ?

Reply:
Yup! The capz monitoring cluster looks for VMs in the CNCF sub that contain this tag